Workflow CA

KristianHaugen98 opened the pull request:
Hi, hope you're all doing well!

Would very much appreciate a feedback on my assignment. 

Thank you so much for your time, and have a nice day! 
Best regards.

KristianHaugen98 committed:
Testing if this works

Just testing if this works, hopefully this ONLY changes MY repo, not for the main repo..

KristianHaugen98 committed:
Satt opp nødvendig forbredelser

* Workflow branch
* Utviklingsverktøy
* Testet isActivePath funksjon

KristianHaugen98 committed:
Ferdig med Workflow CA

Forhåpentligvis er jeg i mål med denne oppgaven, men setter stor pris på tilbakemeldinger fra dere. Takk!

Padletut commented:
The test path in playwright config is not correct, if I run "npm run test:e2e" it is running the /tests/example.spec.js, the config is also missing baseURL.

npx playwright test "./js/testfiles.js/tests.e2e/login.spec.js" failed.

Error: page.goto: Protocol error (Page.navigate): Cannot navigate to invalid URL
    Call log:
      - navigating to "/login", waiting until "load"
      
Affter I added baseURL to playwright config:

Error: page.fill: Unexpected token "username" while parsing selector ""input[name="username"]"
    Call log:
      - waiting for "input[name="username"]

npx playwright test "./js/testfiles.js/navigation.spec.js" also failed.

Error: page.waitForSelector: Test timeout of 30000ms exceeded.
    Call log:
      - waiting for locator('.venue-list') to be visible

KristianHaugen98 committed:
Fikset "navigation to venue details"

Endelig fått fikses slik at nå kjøres denne teste, uten feil!

KristianHaugen98 committed:
login.spec.js

Nå skal denne være i orden, testene fungerer og kjøres igjennom uten problemer.

KristianHaugen98 committed:
isActivePath function

Oppdatert slik at isActivePath funksjonen og testene skal være slik som oppgaven ønsker.

KristianHaugen98 committed:
Playwright og Vitest

Nå fungerer både Playwright og Vitest testene, eg får null feil, og har implementert, slik at det nå inneholder slik oppgaven "Workflow" ber om.

KristianHaugen98 committed:
Oppdaterte README.md filen

Oppdaterte og lagde forbedringer i READEME.md filen.
